bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -542,6 +542,22 @@
 		<config-description-ref uri="thing-type:boschshc:device"/>
 	</thing-type>
 
+
+	<thing-type id="presence-simulation">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="shc"/>
+		</supported-bridge-type-refs>
+
+		<label>Presence Simulation</label>
+		<description>Simulates that someone is home.</description>
+		<semantic-equipment-tag>Application</semantic-equipment-tag>
+
+		<channels>
+			<channel id="presence-simulation-enabled" typeId="presence-simulation-enabled"/>
```

Comment:
@jlaur @lsiepel please let me know if there is a suitable system channel that could be used instead. This is an internal service that can simply be enabled and disabled.

Reply:
I don't think there is, but you can have a look here: https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java